test-integration/operator-tests/src/main/java/org/infinispan/util/CleanUpValidator.java

```diff
@@ -20,7 +20,9 @@ public CleanUpValidator(OpenShift openShift, String appName) {
       conditions = new ArrayList<>();
       conditions.add(
             () -> openShift.apps().statefulSets().withName(appName).get() == null &&
+                  openShift.secrets().withName(appName + "-generated-admin-secret").get() == null &&
```

Comment:
Secret name has been changed, @Crumby please rename it in the validation

Reply:
Done